miqa/learning/nn_training.py

```diff
@@ -399,10 +399,10 @@ def create_train_and_test_data_loaders(df, count_train):
     motion = CustomMotion(p=0.2, degrees=5.0, translation=5.0, num_transforms=1)
     inhomogeneity = CustomBiasField(p=0.1)
     spike = CustomSpike(p=0.1, num_spikes=(1, 1))
-    gamma = CustomGamma(p=0.1)
+    gamma = CustomGamma(p=0.1)  # after quick experimentation: gamma does not appear to help
```

Comment:
Since `gamma` is unused after this, it may be clearer if you comment out the whole line